apps/desktop/src/lib/gitHost/github/githubPrService.ts

```diff
@@ -19,16 +19,21 @@ export class GitHubPrService implements GitHostPrService {
 		private upstreamName: string
 	) {}
 
-	async createPr(title: string, body: string, draft: boolean): Promise<PullRequest> {
+	async createPr(
+		title: string,
+		body: string,
+		draft: boolean,
+		templatePath: string = '.github/PULL_REQUEST_TEMPLATE.md'
```

Comment:
Let's use a constant instead of parameter until a parameter is needed? If you see this signature you'll assume there is e.g.  more than one caller.